src/main/java/it/gov/pagopa/fdr/service/PaymentService.java

```diff
@@ -264,117 +265,42 @@ public GenericResponse deletePaymentFromExistingFlow(
         .build();
   }
 
+  @SneakyThrows
```

Comment:
For which exception was added?

Reply:
This is used in order to avoid Exception-catch on method level (that the compiler tells and pretend to be inserted for correct compilation). This avoid an explicit catch that could possibly avoid automatic DB transaction rollback (via `@Transactional` annotation).